src/main/resources/META-INF/mods.toml

```
@@ -0,0 +1,32 @@
+modLoader="javafml"
+loaderVersion="${loaderVersionRange}"
+license="GNU LGPLv3"
+issueTrackerURL="https://github.com/The-Fireplace/Simply-Caterpillar/issues"
```

Let's change the github links. The old ones technically work but we probably shouldn't rely on them forever.
```suggestion
issueTrackerURL="https://github.com/The-Fireplace-Minecraft-Mods/Simply-Caterpillar/issues"
```

---

Change made in [commit 3ce1a1f](https://github.com/The-Fireplace-Minecraft-Mods/Simply-Caterpillar/pull/4/commits/3ce1a1fe328b2b26d9d65c87b37920deedae9bca)